package.json

```
@@ -1,12 +1,13 @@
 {
-  "name": "@nuxtjs/google-adsense",
-  "version": "2.1.0",
+  "name": "@kylegl/google-adsense",
+  "version": "0.0.1",
```

```suggestion
  "version": "2.1.0",
```

will be bumped automatically with changelog gen in a bit